layouts/partials/help.html

```diff
@@ -3,25 +3,33 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h2 class="section-heading">{{ with .Site.Params.help.title }}{{ . }}{{ end }}</h2>
-                <h3 class="section-subheading text-muted">{{ with .Site.Params.help.subtitle }}{{ . | markdownify }}{{ end }}</h3>
```

Comment:
Yeah, it should end up looking something like this 
![image](https://cloud.githubusercontent.com/assets/730506/22407722/a625ff34-e631-11e6-9ef7-c1200c1f138c.png)